mpc_follower_wrapper/launch/mpc_follower_wrapper.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<launch>
+    <remap from="final_waypoints" to="/guidance/carma_final_waypoints"/>
```

Comment:
Since this launch file is included in guidance.launch this can just be 
<remap from="final_waypoints" to="carma_final_waypoints"/>

The /guidance namespace is applied at runtime